### ISSUE TITLE
Anonymous function in RuleFactory.php throws error

### ISSUE THREAD

**Issue by alberthaff** [nuwave/lighthouse#573]
**Describe the bug**

The anonymous function in `RuleFactory.php:251` throws an error, since it does not allow `$node = null`.

**Expected behavior**

The function should allow `$node = null`. The function already contains logic that accounts for `$node = null`.

**Schema**

```graphql
# Add in relevant parts of your schema definition
```

**Output/Logs**

<details><summary>Click to expand</summary>

```
Argument 1 passed to Nuwave\\Lighthouse\\Schema\\Factories\\RuleFactory::Nuwave\\Lighthouse\\Schema\\Factories\\{closure}() must be an instance of GraphQL\\Language\\AST\\Node, null given
```

</details></br>

**Environment**

Lighthouse Version: 2.6.3
Laravel Version: 5.6.26
PHP Version: .7.2.3

**Additional context**

**Comment by spawnia** [comment]
Can you try `dev-master` and check if your issue persists?

**Comment by alberthaff** [comment]
The issue is fixed in `dev-master`.

**Comment by spawnia** [comment]
Resolved in `v2.6.4` by #582
